systemd/vast.service

```diff
@@ -0,0 +1,45 @@
+[Unit]
+Description=VAST - Visibility Across Space and Time
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+# TODO: move to Type=notify as soon as VAST supports native init systems.
+Type=simple
+
+# user + privileges
+User=vast
+Group=vast
```

Comment:
```suggestion
# We explicitly don't use DynamicUser because that can lead to recursive `chown`s.
# Doing that is pretty slow on some file systems (e.g., xfs).
User=vast
Group=vast
```